easybuild/easyconfigs/q/Quandl/Quandl-3.6.1-foss-2021a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'Quandl'
+version = '3.6.1'
+
+homepage = 'https://pypi.python.org/pypi/Quandl'
+description = "A Python library for Quandl’s RESTful API."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('inflection', '0.5.1', {
+        'checksums': ['1a29730d366e996aaacffb2f1f1cb9593dc38e2ddd30c91250c6dde09ea9b417'],
+    }),
+    ('more-itertools', '8.8.0', {
+        'checksums': ['83f0308e05477c68f56ea3a888172c78ed5d5b3c282addb67508e7ba6c8f813a'],
+    }),
```

Comment:
```suggestion
```

Does this require `more-itertools` 8.8.0? Python 3.9.5 provides `more-itertools` 8.7.0.